sbt-scalajs-bundler/src/main/scala/scalajsbundler/Stats.scala

```diff
@@ -91,17 +110,28 @@ object Stats {
   implicit val assetsReads: Reads[Asset] = (
     (JsPath \ "name").read[String] and
     (JsPath \ "size").read[Long] and
-    (JsPath \ "emitted").readNullable[Boolean] and
-    (JsPath \\ "chunkNames").read[List[String]]
```

Comment:
I suppose this is a breaking change in the `stats` output? If so it would be hard to simultaneously support webpack 4 and 5

Reply:
Yes it is. To reduce the complexity this PR only support webpack 5 format (it was discussed as part of https://github.com/scalacenter/scalajs-bundler/issues/350)